index.html

```diff
@@ -261,6 +261,24 @@ <h2 id="connect" class="connect-title">LET’S CONNECT</h2>
       </ul>
     </div>
   </section>
+  <section class="Form">
+    <div class="f-txt">
+      <h1 class="f-tittle">Contact me</h1>
+      <p class="f-paragraph">If you have an application you are interested in developing, a feature that you need
+        built or a project that needs coding.
+        I’d love to help with it!</p>
+    </div>
+    <form action="https://formspree.io/f/moqzkbbq" method="POST" class="fields">
+      <label for="name-input"><input type="text" name="name" id="name-input" maxlength="30" class="Input"
+          placeholder="Enter your name.." required></label>
+      <label for="email-input"><input type="email" name="email" id="email-input" class="input-email"
+          placeholder="Enter your email.." required></label>
+      <textarea class="message-field" cols="10" rows="8" maxlength="500"
+        placeholder="Write your message here..."></textarea>
+      <button class="b-submit" type="submit"></button>
+    </form>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest sticking to only `input` fields with `placeholder` rather than wrapping it around `label` tags per the [Figma Design Temple](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=VE4s3C9L3c3NruOa-0). 😃 
- [ ] This [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) provides a better understanding of `label` tags. 😃